File: app/src/main/java/com/twismart/wallpapershd/ui/wallpaper/WallpapersContract.java
package com.twismart.wallpapershd.ui.wallpaper;

import com.twismart.wallpapershd.ui.base.BasePresenter;
import com.twismart.wallpapershd.ui.base.BaseView;

/**
 * Created by sneyd on 5/16/2017.
 **/

public class WallpapersContract {

    public interface Presenter<V extends WallpapersContract.View> extends BasePresenter<V> {
        void setWallpaper(String url);
    }

    public interface View extends BaseView {

    }
}

File: app/src/main/kotlin/com/twismart/wallpapershd/ui/wallpaper/WallpapersContract.kt
package com.twismart.wallpapershd.ui.wallpaper

import com.twismart.wallpapershd.ui.base.BaseContract

/**
 * Created by sneyd on 5/16/2017.
 **/

class WallpapersContract {

    interface Presenter<V : WallpapersContract.View> : BaseContract.Presenter<V> {
        fun setWallpaper(url: String)
    }

    interface View : BaseContract.View
}